contribs/gmf/externs/gmf-themes.js

```diff
@@ -542,3 +542,35 @@ gmfThemes.GmfSnappingConfig.prototype.tolerance;
  * @type {boolean|undefined}
  */
 gmfThemes.GmfSnappingConfig.prototype.vertex;
+
+
+/**
+ * @record
+ * @struct
+ */
+gmfThemes.GmfLayerAttributeValuesResponse = function() {};
+
+
+/**
+ * @type {Array.<gmfThemes.GmfLayerAttributeValue>}
```

Comment:
In the next version of Google closure compiler the `?` or the `!` will be required, I thing that's good to add them from now :-)